app/presenters/tree_builder_chargeback_rates.rb

```diff
@@ -35,6 +35,24 @@ def x_get_tree_roots(count_only, options)
         end
         objects
       end
+    when :cb_rates
+      # the rate accordion merge the compute rate and the storage rate in one unic rate grouping by description
+      rates = ChargebackRate.all
+      grouped_rates = rates.group_by(&:description)
```

Comment:
@h-kataria  thanks by your observations. Like I explained in the first headland of this pull request doc, the main idea here is remove the separation (in visual way) between Compute and Storage rates. In one Rate we have all the possible metrics (Compute and Storage) with rate values different from 0. When it is shown in the rate tree two rates (Default-1, Default-2) in the database it is recorded four rates Default-1 and Default-2 of Compute and  Default-1 and Default-2 of Storage. So I thought to group by description.

What we want to achieve with this is to simplify the process of setting up a rate such that a rate should have the rate details that are all valid and leave the door open so it can add new levels of charging, not only Compute and Storage (like Envelope, OpenShift ...)

I know that the correctly here it is change all the model and adapt it to the new requirements but the problem is the assignment tree. The assignment module also follow the separation between Compute and Storage rates.  The process for recollect C&U data and make a Report with this data is more complex and we think attack it in the futute when we have more experience with the Chargeback.

Then we thought that merge Compute and Storage rate only in a visual way it would be a good solution that not affect the assignment and the report process. From yesterday's conversation with @gtanzillo and other manageiq architects, we  think to reconsider this ideas with your considerations. We are open to all suggestions and lines of development that you can do us in these areas.